frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleash.tsx

```diff
@@ -93,10 +98,33 @@ export const NewInUnleash = ({
         'new-in-unleash-seen:v1',
         new Set(),
     );
-    const { isOss, isEnterprise } = useUiConfig();
+    const { isEnterprise } = useUiConfig();
     const signalsEnabled = useUiFlag('signals');
+    const improvedLifecycleEnabled = useUiFlag('lifecycleImprovements');
 
     const items: NewInUnleashItemDetails[] = [
+        {
+            label: 'Lifecycle 2.0',
+            summary: 'Track progress of your feature flags',
+            icon: <MonitorHeartIcon color='primary' />,
+            preview: (
+                <StyledImg
+                    src={LifecycleStagesImage}
+                    alt='Define → Develop → Production → Cleanup → Archived'
+                />
+            ),
+            docsLink:
+                'https://docs.getunleash.io/reference/feature-toggles#feature-flag-lifecycle',
+            show: improvedLifecycleEnabled,
+            longDescription: (
+                <p>
+                    We have updated the names, icons and colours for the
```

Comment:
We have updated the names, icons, and colors for the different stages of a feature flag's lifecycle. The stages convey the same meanings as before but now have clearer names that better indicate where you are in the lifecycle.